### DISCUSSION TITLE
Unable to access the teleport UI.

### DISCUSSION THREAD

**Discussion by phani-hothur** [gravitational/teleport#36911] [Q&A]
We are trying to install teleport on k8s but unable to access the web ui with "secure connection failed" message. Below is the brief on the steps we performed. Anyone have insights on what can be wrong here? 

**About our setup:**

- Going with default, proxy and auth enabled.
- We configured to expose the teleport-proxy through INGRESS with tls enabled.
- Regarding the TLS certificate, we are using the existing one from our namespace(Issuer is letsencrypt), gave same in the "existingSecretName" param. 

**Post installation, we notice:**

-  the Proxy and Auth pods are up. 
-  Ingress is created with provided tls block.
-  Ingress backend points to proxy service
-  Ingress host is DNS added for sure.

**Comment by stevenGravy** [comment]
If you are using multiplex mode then lucky your INGRESS does not support the required TLS routing. https://goteleport.com/docs/architecture/tls-routing/. You may need to use separate mode. What kind of setup is this?
